Readme.md

```diff
@@ -29,9 +29,9 @@ Authors (as of 2015, listed alphabetically by first name):
 
 ## Requirements
 
-* Python 2.7.x or Python 3.4+
+* Python 3.6+
 * Python modules:
-  NumPy, SciPy, Matplotlib
+  NumPy, SciPy, SymPy, Matplotlib, pycddlib
```

Comment:
would it make sense to split between required and optional or do you think we should always require pycddlib?

Reply:
We can make pycddlib optional if you want. On a related note, your tester doesn't yet have pycddlib installed...

Reply:
We can just remove my jenkins instance, I would think. Objections?